features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/Workflow/APIStateChange.jsx

```diff
@@ -0,0 +1,471 @@
+/* eslint-disable react/jsx-props-no-spreading */
```

Comment:
Why we have disabled this rule ??

Reply:
it is disabled because to avoid "Prop spreading is forbidden" issue. I have created certain  props and const and pass them into ContentBase